test/e2e/autoscaling/cluster_size_autoscaling.go

```diff
@@ -653,7 +653,7 @@ var _ = SIGDescribe("Cluster size autoscaling [Slow]", func() {
 		ReserveMemory(f, "memory-reservation", extraPods, totalMemoryReservation, false, defaultTimeout)
 
 		// Apparently GKE master is restarted couple minutes after the node pool is added
-		// reseting all the timers in scale down code. Adding 5 extra minutes to workaround
+		// resetting all the timers in scale down code. Adding 5 extra minutes to workaround
```

Comment:
Can you also fix this same typo that is occuring in two other places: https://github.com/kubernetes/kubernetes/search?q=reseting&unscoped_q=reseting?

Reply:
fix it. plz review it